packages/consent-manager/index.js

```diff
@@ -47,7 +46,11 @@ export default class ConsentManager extends Component {
   }
 
   shouldRequireConsent() {
-    // 1. Check cookies
+    // 1. If prop override is set, always show the consent bar.
+    if (this.props.forceShow) {
+      return true
+    }
+    // 2. Check cookies and apply existing consent preferences if they've already been set.
```

Comment:
Moved this check to the top - I actually believe this was a bug that caused the prop not to work if you'd set consent preferences previously.